vms/avm/service_test.go

```diff
@@ -1555,27 +1555,27 @@ func TestServiceGetTxJSON_OperationTxWithPropertyFxMintOpMultiple(t *testing.T)
 func newAvaxBaseTxWithOutputs(t *testing.T, genesisBytes []byte, chainID ids.ID, fee uint64, parser txs.Parser) *txs.Tx {
 	avaxTx := getCreateTxFromGenesisTest(t, genesisBytes, "AVAX")
 	key := keys[0]
-	tx := buildBaseTx(avaxTx, chainID, fee, key)
+	tx := buildTestBaseTx(avaxTx, chainID, fee, key)
```

Comment:
these renamings here avoid conflicts with functions defined in tx_builders.go. We will further simplify this test code in next PR, https://github.com/ava-labs/avalanchego/pull/2736